v22.2/backup-and-restore-monitoring.md

```diff
@@ -0,0 +1,80 @@
+`schedules_backup_failed` | The number of scheduled backup jobs that have failed.
+`schedules_round_reschedule_wait` | The number of schedules that were rescheduled due to a currently running job. A value greater than 0 indicates that a previous backup was still running when a new scheduled backup was supposed to start. This corresponds to the [`on_previous_running=wait`](create-schedule-for-backup.html#on-previous-running-option) schedule option.
+`schedules_round_reschedule_skip` | The number of schedules that were skipped due to a currently running job. A value greater than 0 indicates that a previous backup was still running when a new scheduled backup was supposed to start. This corresponds to the [`on_previous_running=skip`](create-schedule-for-backup.html#on-previous-running-option) schedule option.
+`jobs_backup_currently_running` | The number of backup jobs currently running in `Resume` or `OnFailOrCancel` state.
+`jobs_backup_fail_or_cancel_retry_error` | The number of backup jobs that failed with a retryable error on their failure or cancelation process.
+`jobs_backup_fail_or_cancel_completed` | The number of backup jobs that successfully completed their failure or cancelation process.
+`jobs_backup_fail_or_cancel_failed` | The number of backup jobs that failed with a non-retryable error on their failure or cancelation process.
+`jobs_backup_resume_failed` | The number of backup jobs that failed with a non-retryable error.
+`jobs_backup_resume_retry_error` | The number of backup jobs that failed with a retryable error.
+`jobs_restore_resume_retry_error` | The number of restore jobs that failed with a retryable error.
+`jobs_restore_resume_completed` | The number of restore jobs that successfully resumed to completion.
+`jobs_restore_resume_failed` | The number of restore jobs that failed with a non-retryable error.
+`jobs_restore_fail_or_cancel_failed` | The number of restore jobs that failed with a non-retriable error on their failure or cancelation process.
+`jobs_restore_fail_or_cancel_retry_error` | The number of restore jobs that failed with a retryable error on their failure or cancelation process.
+`jobs_restore_currently_running` | The number of restore jobs currently running in `Resume` or `OnFailOrCancel` state.
+
+## Datadog integration
+
+To use the Datadog integration with your **{{ site.data.products.dedicated }}** cluster, you can:
+
+- Export the following schedule backup metrics to Datadog using the [Cloud API](../cockroachcloud/cloud-api.html). To set this up, see [Export Metrics From a CockroachDB Dedicated Cluster](../cockroachcloud/export-metrics.html?filters=datadog-metrics-export).
+- Access the Cloud Console **Monitoring** page to enable the integration. To set this up, see [Monitor CockroachDB Dedicated with Datadog](../cockroachcloud/monitoring-page.html#monitor-cockroachdb-dedicated-with-datadog).
+
+### Available metrics in Datadog
+
+Metric | Description 
+-------+-------------
```

Comment:
This metrics list will also need the same adjustment.

Reply:
As per earlier comment, leaving as-is for now! Thanks!